assets/resourcecenter/less/resourcecenter.less

```diff
@@ -2,7 +2,7 @@
 /* ResourceCenter */
 
 #resourcecenter {
-    .resourcedescription {
+    .hobbydescription {
```

Comment:
Any reason for this name change? 
* If they are going to have different styles, use different names. 
* If it's the same style across both, create a common name.